.rubocop.yml

```diff
@@ -0,0 +1,4 @@
+Documentation:
+   Enabled: false
+LineLength:
+   Max: 99
```

Comment:
One small comment about this: you can disable cops here so it doesn't check for a certain thing. Use that with caution, always fix stuff before disabling a check, but keep in mind you can do it (for example, for ugly stuff like copy-pasted code from a library). Also, try to ignore cops for certain blocks of code instead of project-wide (google that)